backend/src/apiserver/storage/list_util.go

```diff
@@ -15,77 +15,43 @@
 package storage
```

Comment:
it looks like @neuromage has moved the entire list util to apiserver/list. could you use that file instead, and delete this file if nothing is used anymore?